security/oath-toolkit/Portfile

```diff
@@ -19,21 +20,36 @@ long_description    The OATH Toolkit contains a shared library, command line \
                     the time-based TOTP algorithm. OATH is the Open \
                     AuTHentication organization which specify the algorithms.
 
-homepage            https://www.nongnu.org/oath-toolkit/
+homepage            http://oath-toolkit.nongnu.org/
 master_sites        savannah
 
 checksums           rmd160  b1fb77b34fdf2459d0899a9fcb8e33f9b9485097 \
                     sha256  cafdf739b1ec4b276441c6aedae6411434bbd870071f66154b909cc6e2d9e8ba \
                     size    4706950
 
+patch.pre_args-replace  -p0 -p1
+
+# Need to regenerate autotools scripts, due to previous patch; not needed for next release
+use_autoreconf      yes
+autoreconf.args     -fvi
+
+# This version of 'oath-toolkit' requires 'xmlsec-1.2'.
+# Once upstream releases support for xmlsec 1.3+, none of this will be needed.
```

Comment:
This would be a good place to add the URL to the upstream bug report tracking xmlsec 1.3 support so that we can check on it in the future to see whether this workaround is still needed.